### ISSUE TITLE
Support for "proxy_protocol"

### ISSUE THREAD

**Issue by aabbccgg** [AdguardTeam/AdGuardHome#5694]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

#### What problem are you trying to solve?
When I using nginx for the frontend & adh's docker for backend, I cannt receive the real ip from the viewer but the local ip instead.
#### Proposed solution
If adh's docker support the proxy_protocol, then I can use this protocol to get the real viewer ip from header

**Comment by fernvenue** [comment]
You can use `X-Forwarded-For` header to do that.

**Comment by fernvenue** [comment]
Hi @L8X, and yea, I'm also pretty sure about that, but this is a duplicate issue of https://github.com/AdguardTeam/AdGuardHome/issues/2798, and I mean we can use `X-Forwarded-For` for now before we can use proxy protocol to get real IP addresses.

**Comment by fernvenue** [comment]
> you cannot use X-Forwarded-For for anything other than HTTP(S) connections.

Calm down dude, that's what I'm talking about, we can use `X-Forwarded-For` at least for HTTP(s) connection for now before AdGuardHome have fully support proxy protocol, and this is a duplicate issue right? I don't think we need to discuss the necessity and technical details of the proxy protocol here.

**Comment by ainar-g** [comment]
Duplicate of #2798

**Comment by aabbccgg** [comment]
thx a lot, I will try it later.
